### DISCUSSION TITLE
Custom FBW: flight deck colours and surfaces

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#3569] [Q&A] [answered]
Hi, my first posting was deleted from "Issues", that's ok. I hope this is the right place to suggest some
flight deck colour and interior surface improvments when using the "Custom FBW 9e6857d":

1) Instrument panel and pedestal panel has two different shades of "Airbus blue" colour
2) Display panels has little bit too much "milky" haze surface
3) White index mark, on the side of the throttle handle, is missing again
4) Most gray switches has an unatural look, like made of almost clear plastik, for ex:
top of some switches looks "bright" gray, and bottom of same switches has too much dark shadows

**Comment by ExampleWasTaken** [comment]
The custom FBW version is WIP and gets updated regularly with all updates the dev version receives and is not supported at the moment as it's experimental. Suggestions should always be for the development version.

**Comment by hotshotp** [comment, marked as answer]
This is already in the development version and will get merged into custom flybywire soon - your issue was closed for a reason. Flybywire is a few commits behind and has the development version texture changes. You are running on a two week old commit.

**Reply by Mickepilot** [reply]
Ok, thank's for your feedbacks. I have the Thrustmaster Airbus TCA Quadrant, so therefore only using the Custom FBW version.
I find it relative hard to follow (and also make) questions, feedbacks, suggestions etc. Not only here at github.com/flybywiresim/a32nx/ but also in all the MS FS Asobo Forums.

**Reply by hotshotp** [reply]
> Ok, thank's for your feedbacks. I have the Thrustmaster Airbus TCA Quadrant, so therefore only using the Custom FBW version.
> I find it relative hard to follow (and also make) questions, feedbacks, suggestions etc. Not only here at github.com/flybywiresim/a32nx/ but also in all the MS FS Asobo Forums.

I suggest redownloading the version ever 3 days or so, it updates a lot every day

**Reply by CptBayer** [reply]
> 
> 
> Ok, thank's for your feedbacks. I have the Thrustmaster Airbus TCA Quadrant, so therefore only using the Custom FBW version.
> I find it relative hard to follow (and also make) questions, feedbacks, suggestions etc. Not only here at github.com/flybywiresim/a32nx/ but also in all the MS FS Asobo Forums.

I also use the TCA Quadrant. Why is it not working in the development mod. What is the difference to the custom mod? 


thans Klaus

**Reply by tracernz** [reply]
It works fine in the stable releases and main development branch with the default MSFS axis settings, although the MCT/FLX detent is a little touchy. The problem is that the throttle response between idle and CL detent is very non-linear due to the way MS made the detents line up. The custom FBW branch allows you to customise the location of the detents so you can use a linear throttle mapping.
